build.ps1

```diff
@@ -226,6 +236,39 @@ function Build-CMakeProject
   Write-Host ""
 }
 
+function Build-WiXProject()
+{
+  [CmdletBinding(PositionalBinding = $false)]
+  param(
+    [Parameter(Position = 0)]
+    [string]$FileName,
+    [hashtable]$Properties = @{}
+  )
+
+  $WiXVersion = "3.11.2"
+  if (-not (Test-Path "$BinaryCache\WiX.$WiXVersion"))
+  {
+    nuget.exe install WiX -Version $WiXVersion -Source https://api.nuget.org/v3/index.json -OutputDirectory $BinaryCache
```

Comment:
I think that this is fine for checkpointing, but I think that we want to integrate that into the wixproj files themselves, so you can just do a `-t:restore` with the project before we do the build.